src/assets/data/character/defender.yml

```diff
@@ -999,4 +999,137 @@
       *Normal > Nether Strike > Bloodstorm Revolution > Nether Strike (100AP damage rotation)*
       
       *Normal > Nether Strike > Flying Guillotine > Nether Strike > Nether Strike (104AP damage rotation)*
+      
+- name: Rain
+  cat: gl
+  rating: 10
+  picture: rain
+  weapon: ohs
+  ace: true
+  limited: true
+  star: 5
+
+  stats:
+    hp: 20580
+    atk: 2884
+    int: 1549
+    def: 2546
+    hit: 819
+    grd: 1022
+
+  talents:
+    - name: "Knight's Honor"
+      effects: 
+        - desc: HP +30%
+          all: true
+          meta:
+            priority: 1
+            buff: HP
+            buffValue: 30
+        - desc: Will not flinch to damage under 20% max HP
+          all: true
+    - name: "Heart on Fire"
+      effects: 
+        - desc: Undermine inflicts ATK/INT/DEF-10% debuff on target for 10s.
+          meta:
+            priority: 4
+            debuff: ATK/INT/DEF
+            buffValue: 10
+    - name: "Young & Skilled"
+      effects: 
+        - desc: Single Target damage +20%
+          all: true
+          meta:
+            priority: 1
+            buff: Single Target Damage
+            buffValue: 40
+        - desc: Taunt+3
+          meta:
+            priority: 1
+            buff: Taunt
+            buffValue: 3
+    - name: "Knight of Grandshelt"
+      effects:
+        - desc: ATK+20%
+          all: true
+          meta:
+            priority: 4
```

Comment:
should be priority 1 (party wide)

Reply:
If that is for Knight of Grandshelt that is a self buff, it only applies to him according to discord and the spreadsheet. It'll apply to him on his stat screen.